### ISSUE TITLE
Feature Request: Add a `hooks` / `events` command that will show all the events that can be hookes

### ISSUE THREAD

**Issue by doronbehar** [beetbox/beets#2794]
I think this feature is needed so users won't have to look inside the source code of the different plugins and look for a line with `self.register_listener` as explained in [the developers' documentation](https://beets.readthedocs.io/en/latest/dev/plugins.html#listen-for-events) in order to know the names of the events they can attach hooks as explained [here](https://beets.readthedocs.io/en/latest/plugins/hook.html).

**Comment by sampsyo** [comment]
Hello! I understand the desire here, but I'm not sure this is technically feasible. There is no central registry of events in the beets dispatch system—they just get invoked (keyed by an arbitrary string) when they occur.

The plugin documentation already links to this list in the docs:
https://beets.readthedocs.io/en/latest/dev/plugins.html#plugin-events

Is that enough?

**Comment by doronbehar** [comment]
OK, I think I didn't totally understood this when I opened the issue. As far as I understand it now, there is a certain set of events that are available for plugins to listen to and for users to listen to using the `hook` plugin, theses events are listed here: https://beets.readthedocs.io/en/latest/dev/plugins.html#plugin-events (The `#plugin-events` doesn't really bring the page to a certain header - it seems like a documentation issue..).

When I posted the issue I thought plugins can create events of their own, meaning the events available for a given configuration depends on the enabled plugins. If it isn't so, than I think just fixing the little issue with the documentation's header `#plugin-events` together with perhaps make the events' names' text bold should be enough to close the issue.
